tests/driver_encx24j600/Makefile

```diff
@@ -5,7 +5,7 @@ FEATURES_REQUIRED = periph_spi periph_gpio
 
 BOARD_INSUFFICIENT_MEMORY := msb-430 msb-430h nucleo32-f031 nucleo32-f042 \
                              nucleo32-l031 nucleo-l053 stm32f0discovery telosb \
-                             z1
+                             wsn430-v3_1b wsn430-v1_4 z1
 
 USEMODULE += gnrc_netdev_default
```

Comment:
This module is removed with the enc28j60 test, but not here

Reply:
Was an accident (but we should do that regardless as a follow-up to #7925, since those test really don't need that module).

Reply:
Fixed (by removing that change from the `enc28j60` test).

Reply:
Also fine with me, as long as it is consistent between the tests.